src/systems/systems/src/electrical/engine_generator.rs

```diff
@@ -300,10 +300,13 @@ mod tests {
             }
         }
     }
-    impl EngineCorrectedN2 for TestEngine {
+    impl EngineN2 for TestEngine {
         fn corrected_n2(&self) -> Ratio {
             self.corrected_n2
         }
+        fn uncorrected_n2(&self) -> Ratio {
+            self.corrected_n2
+        }
```

Comment:
**issue**: An `uncorrected_n2` fn returning `corrected_n2` doesn't make sense. Even though it is a test it still breaks the trait's contract.
**suggestion**: Instead of having one trait, keep the `EngineCorrectedN2` and add `EngineUncorrectedN2`.

Reply:
I was indeed trying not to have multiple traits for related N2 parameters. Can either update the test engine to have a fake uncorrected value even unused as of now, or separate in multiple traits.  Preference for multiple traits then?

Reply:
Definitely. The usages are separated (i.e. engine gen only cares about corrected, the other parts only care about uncorrected) and therefore it's a good idea to let them rely on separate traits as well. If in the future we have usages that require both, we can pass `&(impl EngineCorrectedN2 + EngineUncorrectedN2)`.